codewof/templates/users/activity_graph.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
What is this file used for?

Reply:
This is related to #203 and should be removed from this PR.